subprojects/docs/src/docs/release/notes.md

```diff
@@ -24,6 +24,15 @@ Think of every feature section as a mini blog post.
   NOTE: Totally fine to just link to an example that show the feature.
 -->
 
+### Deprecation warning improvements
```

Comment:
I recommend running prose through https://www.grammarly.com/, I find it's suggestions very useful.

Reply:
Thank you, it's useful!